datastore/data-loader/client_knmi_station.py

```diff
@@ -58,42 +84,52 @@ def netcdf_file_to_requests(file_path: Path | str) -> Tuple[List, List]:
             station_slice = file.sel(station=station_id)
 
             for param_id in knmi_parameter_names:
-                # print(station_id, param_id)
                 param_file = station_slice[param_id]
                 standard_name, level, function, period, period_as_seconds = generate_parameter_name(
-                    (param_file.standard_name if "standard_name" in param_file.attrs else "placeholder"),
+                    (param_file.standard_name if "standard_name" in param_file.attrs else None),
                     param_file.long_name,
                     station_id,
                     station_name,
                     param_id,
                 )
+
+                # Drop parameters that are not CF compliant
+                if not standard_name or standard_name in ["precipitation_duration", "rainfall_duration"]:
+                    continue
+
                 platform = f"0-20000-0-{station_id}"
 
                 ts_mdata = dstore.TSMetadata(
                     platform=platform,
                     instrument=param_id,
                     platform_name=station_name,
                     title=param_file.long_name,
+                    license="CC BY 4.0",
                     standard_name=standard_name,
-                    unit=param_file.units if "units" in param_file.attrs else None,
+                    unit=convert_unit_names(param_file.units) if "units" in param_file.attrs else None,
                     level=level,
                     period=period_as_seconds,
                     function=function,
                     parameter_name=":".join([standard_name, str(float(level / 100)), function, period]),
                     naming_authority="nl.knmi",
                     keywords=file["iso_dataset"].attrs["keyword"],
+                    summary=file["iso_dataset"].attrs["abstract"],
                     keywords_vocabulary=file.attrs["references"],
                     source=file.attrs["source"],
                     creator_name="KNMI",
                     creator_email=file["iso_dataset"].attrs["email_dataset"],
                     creator_url=file["iso_dataset"].attrs["url_metadata"],
-                    creator_type="Institution",
+                    creator_type="institution",
```

Comment:
Is this actually an Enum in the ingestor?

Reply:
Yes, it is defined as `creator_type: Optional[Literal["person", "group", "institution", "position"]]` in the `model`.